### PR TITLE
program.txt is added

### PR DESCRIPTION
**Pull request by harpreet-nerd**
@ajabble i have added program.txt file please check ,is it ok now?

### CODE REVIEW

File: program.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+define void factorial(int n)
```
Comment (ajabble) [inline program.txt]: What do you mean by define? void?

File: program.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+int fact =1;
```
Comment (ajabble) [inline program.txt]: space after assignment operator

File: program.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+for(i=i;i<=n;i++)
```
Comment (ajabble) [inline program.txt]: Please learn first google coding best practices thereafter you will let know yourself how to write code.

File: program.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+fact = fact*i;
+printf("%d",fact);
```
Comment (ajabble) [inline program.txt]: Proper log statement

File: program.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+define void factorial(int n)
+int fact =1;
+for(i=i;i<=n;i++)
+fact = fact*i;
```
Comment (ajabble) [inline program.txt]: google best practices part

File: program.txt [hunk 6]
```diff
@@ -0,0 +1,11 @@
...
+printf("%d",fact);
+
+define evenOdd(int num)
+if(num%2 == 0)
```
Comment (ajabble) [inline program.txt]: Google best practices part

**Comment by ajabble** [comment]
Also, please re-indent your code before you push.

File: program.txt (hunk 1 above)
Comment (harpreet-nerd) [reply program.txt]: define is preprocessor directive and void is data type of factorial function

**Comment by ajabble** [comment]
It should not be closed. You should attend feedback on same PR.

**Pull request closed without merging**